modules/ocf/manifests/auth.pp

```diff
@@ -132,6 +132,14 @@
     require => Augeas['sshd_config'],
   }
 
+  # Export SSH keys from every host, and use them to populate the global list
+  # in /etc/ssh/ssh_known_hosts
+  @@sshkey { $::hostname:
+    type => ecdsa-sha2-nistp256,
+    key  => $::sshecdsakey,
+  }
+  Sshkey <<| |>>
```

Comment:
I'm hoping this will remove the errors on regenerating a machine, but I don't think it will since each user stores this information in their local `~/.ssh` directory. It will make it so that `yes` doesn't have to be typed for the first connection though, so that's nice.